docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -156,6 +156,10 @@ DKIM is currently supported by either OpenDKIM or Rspamd:
         use_esld = true;
         check_pubkey = true; # you want to use this in the beginning
 
+        selector = "mail";
+        path = "/tmp/docker-mailserver/opendkim/keys/$domain/$selector.private"; # this will automatically match keys for domains
```

Comment:
~~Can you please cite the source for this feature?~~ (_**EDIT:** Not necessary, cited below_)

I see on the [rspamd dkim docs](https://rspamd.com/doc/modules/dkim_signing.html#configuration) that `selector` is used as the default with the `$selector` variable, ~~but `$domain` is only mentioned in a comment, is that also a fixed fallback value or is it documented somewhere as dynamic?~~

**EDIT:** After reading the [first section](https://rspamd.com/doc/modules/dkim_signing.html#principles-of-operation) and going over the config example / defaults, I see that `$domain` is from `use_domain` setting, which defaults to dynamically sourced from `header` but could be configured as a fixed domain. While `selector` defaults to `dkim`.

Reply:
This context with source citing is now also part of the related TODO issue: https://github.com/docker-mailserver/docker-mailserver/issues/3778